apps/docs-snippets/src/guide/encoding/encode-and-decode.test.ts

```diff
@@ -52,18 +52,21 @@ describe('encode and decode', () => {
     // #endregion encode-and-decode-3
 
     // #region encode-and-decode-4
-    // #import { JsonAbiArgument, AbiCoder};
+    // #import { Interface };
 
     // Now we can encode the argument we want to pass to the function. The argument is required
-    // as a function parameter for all `AbiCoder` functions and we can extract it from the ABI itself
-    const argument: JsonAbiArgument = abi.functions
+    // as a function parameter for all abi functions and we can extract it from the ABI itself
+    const argument = abi.functions
       .find((f) => f.name === 'main')
-      ?.inputs.find((i) => i.name === 'inputted_amount') as JsonAbiArgument;
+      ?.inputs.find((i) => i.name === 'inputted_amount')?.concreteTypeId as string;
 
-    // Using the `AbiCoder`'s `encode` method,  we can now create the encoding required for
-    // a u32 which takes 4 bytes up of property space
+    // The `Interface` class is the entry point for encoding and decoding all things abi-related.
```

Comment:
A lot of the documentation [here](https://docs.fuel.network/docs/fuels-ts/encoding/encode-and-decode/) will also need to be changed to be brought in line.

Reply:
Whoops, forgot about that. https://github.com/FuelLabs/fuels-ts/pull/2862/commits/3df8bdabdb243295e3e7a3b00c104fca8048948f